wasm/callback_128.wast

```diff
@@ -1,4 +1,5 @@
 (func $callback_128
+  (export "2")
```

Comment:
I think these exports should be added in the JS part with `(export "2" $callback_128)` so there would be a single place of truth for the indexes.

Reply:
Actually you could make these part of the `exports` list you had a comment about below :)